posts/2021/04/a-step-towards-educating-with-Spyder.md

```diff
@@ -0,0 +1,89 @@
+<!--
+.. title: A step towards educating with Spyder 
+.. slug: a-step-towards-educating-with-Spyder
+.. date: 2021-04-11 08:00:00 UTC-06:00
+.. author: Juanita Gomez
+.. tags: Spyder, community, grant, funding
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+
+As a community manager in the Spyder team, I have been looking for ways of 
+involving more users in the community and making Spyder useful for a larger 
+number of people. With this, a new idea came: Education.
+
+For the past months, we have been wondering with the team whether Spyder 
+could also serve as a teaching-learning platform, especially in this era 
+where remote instruction has become necessary. We submitted a proposal to the 
+Essential Open Source Software for Science (EOSS) program of the Chan 
+Zuckerberg Initiative, during its third cycle, with the idea of providing a 
+simple way inside Spyder to create and share interactive tutorials on topics 
+relevant to scientific research. Unfortunately, we didn’t get this funding, 
+but we didn’t let this great idea die.
+
+<!-- TEASER_END -->
```

Comment:
I suggest moving this to after the next paragraph, to get the PSF grant mention into the teaser.

Reply:
Sounds good @rgommers. Thanks :)